### PR TITLE
Bring Compilers.sln into sync with Roslyn.sln

### PR DESCRIPTION
**Pull request by agocke**
It looks like a couple of changes brought them out of sync
and VS really dislikes it.

/cc @dotnet/roslyn

### CODE REVIEW

**Comment by jcouv** [comment]
Thanks!
:shipit:

**Comment by agocke** [comment]
@jasonmalinowski There's a weird ETA failure that I think is unrelated to my change. Would you like someone to look at it or should I just file a bug?

**Comment by agocke** [comment]
retest windows_eta_open_prtest please

File: src/Dependencies/PooledObjects/Microsoft.CodeAnalysis.PooledObjects.shproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Globals">
-    <ProjectGuid>c1930979-c824-496b-a630-70f5369a636f</ProjectGuid>
```
Comment (jasonmalinowski) [inline src/Dependencies/PooledObjects/Microsoft.CodeAnalysis.PooledObjects.shproj]: It looks like this GUID is still being used in Roslyn.sln and CrossPlatform.sln. The fix might be to update the Compiler.sln version instead?

**Pull request merged**